src/resources/projects/files.ts

```diff
@@ -13,12 +13,12 @@ export class Files extends APIResource {
    */
   create(
     projectId: string,
-    path: string,
+    path_: string,
```

Comment:
Did we rename this parameter or is stainless trying to avoid clobbering some global?

Reply:
Yeah I don't think we merged anything around that time, seems like they're doing it? https://github.com/val-town/sdk/pull/537/files

Bit of a bummer